src/scss/language-processing.scss

```diff
@@ -138,6 +138,82 @@
     }
 }
 
+.media-frame-content {
+	.generated-images,
+	.prompt-view {
+		padding: 0 16px
+	}
+
+	.generated-images {
+		ul {
+			display: flex;
+			flex-wrap: wrap;
+			list-style-type: none;
+			gap: 18px;
+			margin: 0;
+			padding: 0;
+		}
+
+		li {
+			margin-bottom: 20px;
+		}
+
+		.generated-image {
+			img {
+				box-shadow: inset 0 0 15px rgb(0 0 0 / 10%), inset 0 0 0 1px rgb(0 0 0 / 5%);
+				border: 1px solid rgb(0 0 0 / 10%);
+				background: #f0f0f1;
+				display: block;
+				margin-bottom: 10px;
+				width: 460px;
```

Comment:
Since we are adding width 460px, On the mobile devices it will cause overflow scroll issue. Lets add `max-width: 100%;` with it so default width will be 460 and maximum width will be like container width.